public/static/locales/th/elective.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title_student": "เลือกวิชาเลือก",
+  "title_teacher": "วิชาเลือกของคุณ",
+  "title_admin": "วิชาเลือก",
+  "list": {
+    "action": {
+      "choose_initial": "เลือกวิชานี้",
+      "choose_chosen": "เลือกแล้ว",
+      "choose_change": "เปลี่ยนเป็นวิชานี้"
+    }
+  },
+  "detail": {
+    "information": {
+      "title": "รายละเอียด",
+      "name": "ชื่อวิชา",
+      "description": "รายละเอียด",
+      "code": "รหัสวิชา",
+      "teachers": "ครู",
+      "room": "ห้องที่ใช้เรียน",
+      "credit": "หน่วยกิต"
+    },
+    "trade": {
+      "unavailable_future": "ขณะนี้นักเรียนยังไม่สามารถแลกวิชาเลือกได้ ระบบจะเปิดให้แลกวิชาเลือกในวันที่ 24 พ.ค.",
```

Comment:
To me, this should be rewritten without the word "นักเรียน". Since when I read other paragraphs in the app, the wording seems friendly like we're friends.

Reply:
We try to be concise but also personal. But we’re not friends with the user. Our express purpose is to communicate clearly without yapping too much, and that doesn’t really align with friendship.

Check out [Material Design’s guide on content design](https://m3.material.io/foundations/content-design/overview).

Though, what do you prefer it to say instead?